.gitignore

```diff
@@ -1,2 +1,5 @@
 .env
 __pycache__
+migrations/
```

Comment:
The migration files should be committed - when your collaborators pull your code with migration files, they will be able to recreate the db tables exactly as you have them. 

>The migration files for each app live in a “migrations” directory inside of that app, and are designed to be committed to, and distributed as part of, its codebase. You should be making them once on your development machine and then running the same migrations on your colleagues’ machines, your staging machines, and eventually your production machines. ([source](https://docs.djangoproject.com/en/3.2/topics/migrations/))